docs/server/push-duty.rst

```diff
@@ -40,6 +40,40 @@ This way we can ensure that:
 Making multiple releases of a package during a weekly milestone is totally
 fine since this helps with testing smaller sets of changes.
 
+Security Fixes
+++++++++++++++
+
+Security fixes for addons-server live on a separate private repository:
+
+    * `addons-server-security <https://github.com/mozilla/addons-server-security>`_
+
+When making a pull request against this repository, the remote branch should
+not be published to one's fork but to the repository itself, keeping the
+changes entirely private. Once the PR has been reviewed, it should *not* be
+merged right away. Instead, merging to master is part of push duty and happens
+right before tagging::
+
+  $ git checkout master
+  $ git pull
+  $ git fetch security
+  $ git merge security/<branch-name>
```

Comment:
The cool thing about merging is that it should close the pull request on addons-server-security automatically since the commit does not change. I did cherry-pick in the past but this seems cleaner.